.tflint.hcl

```diff
@@ -0,0 +1,5 @@
+plugin "azurerm" {
+  enabled = true
+  version = "0.14.0"
```

Comment:
Update this to 1.x, we shouldn't be maintaining code this old.

Reply:
that is for tflint version, not tf version https://github.com/terraform-linters/tflint-ruleset-azurerm